openslides/core/views.py

```diff
@@ -141,6 +141,18 @@ def get(self, *args, **kwargs):
                 else:
                     js_files.extend(app_js_files)
 
+        client_settings_keys = [
+            # Add new settings to personal settings.py, utils/settings.py.tpl and to this list. Remove this comment later.
```

Comment:
@emanuelschuetze Only add new settings keys (as strings) to this list. Of course add key and value to settings.py and settings.py.tpl.